pkg/controller/seed-controller-manager/mla/mla.go

```diff
@@ -106,7 +107,7 @@ func Add(
 	}
 	// FIXME: Grafana API uses a global user context switches to manage datasources,
```

Comment:
you should be also able to remove the FIXME comment and use numWorkers instead of 1

Reply:
yes, of course. I still plan to add some changes